ament_cmake_cppcheck/cmake/ament_cmake_cppcheck_lint_hook.cmake

```diff
@@ -33,6 +33,12 @@ if(_source_files)
     list(APPEND _all_include_dirs ${ament_cmake_cppcheck_ADDITIONAL_INCLUDE_DIRS})
   endif()
 
+  # Language
+  set(_language "c")
```

Comment:
Before this PR there is no default language and the tool determines the language automatically. That behavior should be preserved. If no language is specified no `LANGUAGE` / `--language` should be passed.